### ISSUE TITLE
"vue/no-multi-spaces" wrongfully triggering error when "$emit()" is found

### ISSUE THREAD

**Issue by luisjs** [vuejs/eslint-plugin-vue#572]
**Tell us about your environment**

* **ESLint Version:** 5.5.0
* **eslint-plugin-vue Version:** next (5.0.0-beta.3)
* **Node Version:** v10.1.0

**Please show your full configuration:**

```
{
	"env": {
		"browser": true,
		"node": true,
		"mocha": true
	},
	"extends": [
		"plugin:vue/recommended",
		"standard"
	],
	"globals": {
		"expect": true
	},
	"overrides": [
		{
			"files": ["*.vue"],
			"rules": {
				"indent": "off",
				"vue/no-v-html": "off",
				"vue/html-indent": ["error", "tab"],
				"vue/script-indent": ["error", "tab", { "baseIndent": 1 }]
			}
		}
	],
	"parserOptions": {
		"ecmaVersion": 2018,
		"parser": "babel-eslint"
	},
	"plugins": [
		"vue"
	],
	"rules": {
		"arrow-parens": ["error", "as-needed"],
		"brace-style": ["error", "stroustrup"],
		"indent": ["error", "tab", { "SwitchCase": 1 }],
		"max-len": ["warn", 140],
		"no-console": "off",
		"no-extra-semi": "error",
		"no-tabs": "off",
		"semi": ["error", "always"],
		"space-before-function-paren": ["error", "never"]
	}
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
	<component
		:is="type"
		:type="action"
		:class="[
			'base-circular-button',
			block ? '-block' : ''
		]"
		@click="$emit('click', $event)"
	>
		<slot />
	</component>
</template>
```

**What did you expect to happen?**
Everything would compile with no errors.

**What actually happened? Please include the actual, raw output from ESLint.**
```
WARNING in ./src/components/base/clickable/BaseCircularButton.vue
Module Error (from ./node_modules/eslint-loader/index.js):

C:\...\src\components\base\clickable\BaseCircularButton.vue
  9:11  error  Multiple spaces found before 'mit(''  vue/no-multi-spaces

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix`
option.

 @ ./src/components sync [\w-]+\.vue$ ./base/clickable/BaseCircularButton.vue
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 ./src/index.js

```
